prawcore/auth.py

```diff
@@ -93,7 +93,7 @@ def revoke_token(self, token, token_type=None):
         if token_type is not None:
             data["token_type_hint"] = token_type
         url = self._requestor.reddit_url + const.REVOKE_TOKEN_PATH
-        self._post(url, success_status=codes["no_content"], **data)
```

Comment:
Seems like this should be 200.

>  The authorization server responds with HTTP status code 200 if the
>   token has been revoked successfully or if the client submitted an
>   invalid token.

Reply:
There isn't a need to specify that status code [because the default is 200.](https://github.com/praw-dev/prawcore/blob/902538be24ca4b575bc1113921deaa6883b52f6b/prawcore/auth.py#L29)

Reply:
Of course. Thanks for pointing that out.